scripts/release_notes/commitlist.py

```diff
@@ -252,6 +280,11 @@ def main():
     group = parser.add_mutually_exclusive_group(required=True)
     group.add_argument('--create_new', nargs=2)
     group.add_argument('--update_to')
+    # I found this flag useful when experimenting with adding new auto-categorizing filters.
+    # After running commitlist.py the first time, if you add any new filters in this file,
```

Comment:
what do you mean by filter here?

Reply:
Just adding new directories that map directly to specific categories. I was experimenting with adding new ones, so this change made it convenient to add a new directory -> category mapping, and see which PR's got newly labeled because of it.